.github/workflows/icfy-stats.yml

```diff
@@ -0,0 +1,44 @@
+name: Calculate ICFY stats
+
+on: [push]
+   
+jobs:
+  build:
+    name: Build ICFY stats
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Set up Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '>=12.15.0'
+    - name: Checkout code
+      uses: actions/checkout@master
```

Comment:
```suggestion
      with:
        fetch-depth: 0
```

Reply:
This is probably a better recipe:

https://github.com/actions/checkout#fetch-all-tags

```yaml
- uses: actions/checkout@v2
- run: |
    git fetch --no-tags --prune --depth=1 origin +refs/heads/master:refs/remotes/origin/master
```